examples/http_server/client/src/main.rs

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use prost::Message;
+use std::{fs, io};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Clone)]
+#[structopt(about = "HTTPS server pseudo-Node Client Example.")]
+pub struct Opt {
+    #[structopt(
+        long,
+        help = "Path to the PEM-encoded CA root certificate.",
+        default_value = "../../certs/local/ca.pem"
+    )]
+    ca_cert: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Send a request, and wait for the response
+    let label = oak_abi::label::Label::public_untrusted();
+    let mut label_bytes = vec![];
+    let _ = label.encode(&mut label_bytes);
```

Comment:
nit: I know it is a pre-existing pattern elsewhere in these files, but it would be clearer to unwrap or use ? rather than to just ignore the result (even though it should never cause an error).